ci/cases/gfsv17/ocnanal.yaml

```diff
@@ -25,6 +26,6 @@ ocnanal:
   NICAS_GRID_SIZE: 15000
 
 prepoceanobs:
-  SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml 
+  SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml
```

Comment:
Should be moved to `config.prepoceanobs`. 
Same in `config.ocnanl`.
Also see comment below.

Reply:
What's wrong with keeping this exposed in the yaml?